Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1669,7 +1669,7 @@ HashDigest<SHA256> blockHash
             }
         }
 
-        private InvalidBlockException ValidateNextBlock(Block<T> nextBlock)
+        private InvalidBlockException ValidateNextBlock(Block<T> nextBlock, bool evaluateActions)
```

Comment:
It is just a suggestion, IMO it should have another naming like `validateStateRootHash` or better to present its purpose.

Reply:
Amended https://github.com/planetarium/libplanet/pull/1017/commits/6de74c53b6275a13ee47e26a7cba86ff15e456aa